src/lib/Form/Type/ChoiceList/Loader/SiteAccessChoiceLoader.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use EzSystems\EzPlatformAdminUi\Siteaccess\NonAdminSiteaccessResolver;
+use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
+use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
+
+class SiteAccessChoiceLoader implements ChoiceLoaderInterface
```

Comment:
You might want to extend `Symfony\Component\Form\ChoiceList\Loader\AbstractChoiceLoader` instead.

Reply:
@Steveb-p there is no such class in Symfony 3.4.

Reply:
It's a moot point then, thanks for checking.